.gitignore

```diff
@@ -6,6 +6,8 @@
 /src/test/javascript/PhantomJS*/
 /uploads
 /test-results.xml
+/src/main/resources/config/application-artemis.yml
+/src/main/resources/config/application-dev.yml
```

Comment:
Please remove them from here and use a local setup to prevent staging these files.

Reply:
i removed these paths/files from the gitignore